stackslib/src/chainstate/nakamoto/tests/node.rs

```diff
@@ -63,6 +65,88 @@ use crate::net::test::{TestPeer, TestPeerConfig, *};
 use crate::util_lib::boot::boot_code_addr;
 use crate::util_lib::db::Error as db_error;
 
+#[derive(Debug, Clone)]
+pub struct TestSigners {
+    /// The parties that will sign the blocks
+    pub signer_parties: Vec<wsts::v2::Party>,
+    /// The commitments to the polynomials for the aggregate public key
+    pub comms: Vec<wsts::common::PolyCommitment>,
```

Comment:
We use the word `comms` elsewhere to refer to "communication channels".  Can this member be renamed to something like `signer_commits` to remove ambiguity?

Reply:
Will do!